dialogue/TownNPCs/Olive/InitialInteractions/AzuraBanana/AzuraBanana_Olive_First2.json

```diff
@@ -25,12 +25,21 @@
       "Me alegro de que estéis todos bien. No sé que haría si algo le pasase a <c=pistachio>mi hermana pequeña</c>...",
       "¡Tened cuidado en vuestras aventuras!"
     ],
+    "pl": [
+      "Nie pamiętasz? Rozumiem czemu, ponieważ nigdy nie rozmawialiśmy bezpośrednio...",
+      "Nazywam się <c=pistachio>Olive</c>. Jestem starszą siostrą <c=pistachio>Pistachii</c>!",
+      "Słyszałam że wy próbowaliście powstrzymać kradzenie kryształów... ale <c=violastro>ten złodziej</c> uciekł, prawda?",
```

Comment:
Use "kradzież" instead of "kradzenie". Also "wy" seems kinda unnecessary here, but it doesn't change that much

Reply:
I wanted to put in 'wy' to emphasise that the character is part of a group, but I can remove it if it would be better.

Reply:
It's pretty much up to you then